docs/csharp/tutorials/console-webapiclient.md

```diff
@@ -126,13 +114,12 @@ Next, rename the namespace defined in the `namespace` statement from its default
 
 Next, update the `Main` method to call this method. The
 `ProcessRepositories` method returns a Task, and you shouldn't exit the
-program before that task finishes. Therefore, you must use the `Wait`
-method to block and wait for the task to finish:
+program before that task finishes. Therefore, you must change the signature of `Main`. Add the `async` modifier, and change the return type to `Task`. Then, in the body of the method, add a call to `ProcessRepositories`. Make sure that you `await` that method call:
```

Comment:
"Make sure that you `await` that method call" -> I'm not sure how well that will translate into other languages while keeping await unlocalized.